src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -203,37 +210,34 @@ public void nonStringAssetRootTest() throws Exception {
     BlobInfo sampleBlob =
         BlobInfo.newBuilder(bucket, targetPath + "ingest-test-sample.json").build();
     BlobInfo fileBlob = BlobInfo.newBuilder(bucket, targetPath + "ingest-test-file.json").build();
+    blobsToDelete.addAll(Arrays.asList(participantBlob, sampleBlob, fileBlob));
```

Comment:
I think `List.of()` is preferred over `Arrays.asList()` for this case
```suggestion
    blobsToDelete.addAll(List.of(participantBlob, sampleBlob, fileBlob));
```